autogluon_zeroshot/simulation/ensemble_selection_config_scorer.py

```diff
@@ -34,10 +33,17 @@ def __init__(self,
                  max_fold: Optional[float] = None,
                  backend: str = 'native',
                  use_fast_metrics: bool = True,
+                 proxy_fit_metric_map: Optional[Union[dict, str]] = 'default',
                  ):
         """
         TODO: Add docstring
         :param use_fast_metrics: If True, will leverage optimized eval metrics to speed up config scoring.
+        :param proxy_fit_metric_map:
+            If eval_metric is among the keys in the `proxy_fit_metric_map` dictionary,
+            the value eval_metric will be used during the weighted ensemble fitting process as a proxy.
+            For example, the proxy metric could be faster to compute while producing a similar end result.
+            If None: Do not use proxy metrics, equivalent to {}.
+            If 'default': set to {'roc_auc': 'log_loss'}, making 'log_loss' a proxy to 'roc_auc'
```

Comment:
This is a bit confusing as we are passing a dict to pretend one metric is like the other one. Can we just allow to configure using `logloss`?

Reply:
We discussed offline on this, as configuring with logloss isn't the same result